_includes/commands/advtable-cmds.md

```diff
@@ -24,4 +18,23 @@ tinymce.activeEditor.execCommand('mceSortTableByColumnAsc')
 tinymce.activeEditor.execCommand('mceSortTableByColumnDesc')
 tinymce.activeEditor.execCommand('mceSortTableByRowAsc')
 tinymce.activeEditor.execCommand('mceSortTableByRowDesc')
-```
\ No newline at end of file
+tinymce.activeEditor.execCommand('mceTableToggleSeries', false, { name: 'numeric', location: 'left' })
+```
+
+### `mceSortTableAdvanced`
+
+`mceSortTableAdvanced` accepts an object with the following key-value pairs:
+
+- `sortby` - `'row'` or `'column'`
+- `roworcol` - a zero-indexed integer in a string representing the row from the top of the table or column from the left of the table.
+- `sort` - `'row'`, `'column'`, `'selection'`, or `'table'`
+- `order` - `'ascending'` or `'descending'`
+
+### `mceTableToggleSeries`
+
+`mceTableToggleSeries` accepts an object with the following key-value pairs:
+
+| Name | Value | Requirement | Description |
+| ---- | ----- | ----------- | ----------- |
+| name | `string` | Required |  Specifies the series to toggle. Series and their associated names are configured using the [advtable_value_series option]({{site.baseurl}}/plugins/premium/advtable/#advtable_value_series). If the series name provided is not present on the table, a series column will be inserted or an existing series column will be replaced by the new series. |
```

Comment:
I think this is right?

```suggestion
| name | `string` | Required |  Specifies the series to toggle. Series and their associated names are configured using the [advtable_value_series option]({{site.baseurl}}/plugins/premium/advtable/#advtable_value_series). If a series column with the series name provided is not present on the table, a series column will be inserted or an existing series column will be replaced by the new series. |
```

Should we explain what happens if there is already a series column with the same name?

Reply:
I am wondering if doing something like this would be too verbose?

```
| Name | Value | Requirement | Description |
| ---- | ----- | ----------- | ----------- |
| name | `string` | Required |  Specifies the series to toggle. Series and their associated names are configured using the [`advtable_value_series` option]({{site.baseurl}}/plugins/premium/advtable/#advtable_value_series). |

If a series column is not present on a table:
- a series column using the specified series will be inserted at the start of the table.

If a series column is present on a table:
- if the specified series matches, the series column is removed from the table.
- if the specified series does not match, the series column is updated to use the new series.
```

Reply:
I like taking the toggling information out of the table, but maybe structure it more like this?

```suggestion
| name | `string` | Required |  Specifies the series to toggle. Series and their associated names are configured using the [advtable_value_series option]({{site.baseurl}}/plugins/premium/advtable/#advtable_value_series). |

If the table already has a series column that uses the series specified in `name`, the series column will be removed from the table. Otherwise, a new series column will be created, replacing any other series column that may already be in the table.
```

Reply:
Yep, I like that suggestion so will use that. Thanks.